client/src/pages/CalendarHomePage/CalendarHomePage.js

```diff
@@ -18,24 +21,59 @@ class CalendarHomePage extends Component {
 
     this.state = {
       newShow: null,
+      selectedDate: moment(),
       shows: {},
     };
   }
 
+  dateChangeHandler = date => {
+    if (date instanceof Date) {
+      this.setState({ selectedDate: moment(date) });
+    } else {
+      this.setState({ selectedDate: date });
+    }
+  };
+
   render() {
     const { showsError } = this.props;
+    const { selectedDate } = this.state;
+
+    // we will set a key for the month/year that is currently selected
+    // so that the displayed month in DayPickerSingleDateController will auto-update
+    // as the selected date property changes
+    // (workaround for this issue: https://github.com/airbnb/react-dates/issues/48)
+    // unfortunately, this also causes the calendar to flash briefly while selecting a date in a different month
+    const dayPickerKey =
+      String(selectedDate.month()) + String(selectedDate.year());
 
     return (
-      <div className="calendar__view">
-        {showsError && (
-          <Alert
-            type="danger"
-            header="Error Loading Shows"
-            text={showsError.response.data.message}
+      <div className="calendar">
+        <div className="calendar__sidebar">
+          <DayPickerSingleDateController
+            key={dayPickerKey}
+            date={selectedDate}
+            horizontalMonthPadding={0}
+            noBorder={true}
+            daySize={28}
+            numberOfMonths={1}
+            onDateChange={this.dateChangeHandler}
+            focused={true}
+            onFocusChange={({ focused }) => this.setState({ focused: true })} // Force the focused states to always be truthy so that date is always selectable
```

Comment:
Component looks good!  We can remove the shortcut menu with ```hideKeyboardShortcutsPanel={true}``` if you want the home page to look a bit cleaner.